### PR TITLE
fix margins header

### PR DESCRIPTION
**Pull request by GusBaamonde**
fix: margins and paddings header
feat: add styles for scrollbar navigation

### CODE REVIEW

File: assets/scss/core/_reset.scss [hunk 1]
```diff
@@ -25,6 +25,22 @@ RESET DOPPLER STYLE GUIDES
...
+  border-radius: 10px;
+}
+
+::-webkit-scrollbar-track {
+  background: rgba(0, 0, 0, 0.1);
+  border-radius: 10px;
+  margin: 5px;
+}
+
+::-webkit-scrollbar {
```
Comment (andresmoschini) [inline assets/scss/core/_reset.scss]: Does this changes only affect webkit? is that fine?

**Comment by andresmoschini** [comment]
Thanks @GusBaamonde 

If I am not wrong there are still pending details, but in my opinion it is enough good for the moment.

image

File: assets/scss/core/_reset.scss [hunk 2]
```diff
@@ -25,6 +25,22 @@ RESET DOPPLER STYLE GUIDES
   box-sizing: border-box;
 }
 
+::-webkit-scrollbar-thumb {
+  background: rgba(0, 0, 0, 0.3);
```
Comment (cbernat) [inline assets/scss/core/_reset.scss]: This color maybe should be set as a variable right?

File: assets/scss/core/_reset.scss (hunk 1 above)
Comment (cbernat) [reply assets/scss/core/_reset.scss]: I agree with this, why are you styling webkit scrollbar only? What about the other browsers? Maybe I'm missing something.

Comment (andresmoschini) [reply assets/scss/core/_reset.scss]: It seems that this _Doppler's issue_ only affects webkit, so, it is fine fixing webkit.

File: assets/scss/core/_reset.scss (hunk 2 above)
Comment (GusBaamonde) [reply assets/scss/core/_reset.scss]: Esto hasta quizás se modifique y no lo vamos a usar en ningún otro lado. Por el momento lo dejaría asi.

**Pull request merged**